dynamic-config.md

```diff
@@ -155,8 +156,10 @@ The following TiKV configuration items can be modified online:
 | `raftstore.cleanup-import-sst-interval` | The time interval to check expired SST files |
 | `raftstore.local-read-batch-size` | The maximum number of read requests processed in one batch |
 | `raftstore.hibernate-timeout` | The shortest wait duration before entering hibernation upon start. Within this duration, TiKV does not hibernate (not released). |
-| `raftstore.apply-pool-size` | The number of apply thread pool size |
-| `raftstore.store-pool-size` | The number of store thread pool size |
+| `raftstore.apply-pool-size` | The number of threads in the pool that flushes data to storage, which is the size of the Apply thread pool |
+| `raftstore.store-pool-size` | The number of threads in the pool that processes Raft, which is the size of the Raftstore thread pool |
+| `raftstore.apply-max-batch-size` | Raft state machines execute data write requests in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can execute the requests in one batch. |
+| `raftstore.store-max-batch-size` | Raft state machines execute requests for log flushing in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can execute the requests in one batch. |
```

Comment:
```suggestion
| `raftstore.store-max-batch-size` | Raft state machines process requests for spilling logs into disk in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can process the requests in one batch. |
```

Reply:
This parts was updated in [#8644](https://github.com/pingcap/docs-cn/pull/8644) and revised in [#8784](https://github.com/pingcap/docs-cn/pull/8784)

Reply:
comment modified